aws/backing-services/rds-replica.tf

```diff
@@ -0,0 +1,179 @@
+variable "rds_replica_name" {
+  type        = "string"
+  default     = "rds-replica"
+  description = "RDS instance name"
+}
+
+variable "rds_replica_enabled" {
+  type        = "string"
+  default     = "false"
+  description = "Set to false to prevent the module from creating any resources"
+}
+
+variable "rds_replica_replicate_source_db" {
+  type        = "string"
+  description = "Specifies that this resource is a Replicate database, and to use this value as the source database. This correlates to the identifier of another Amazon RDS Database to replicate. Note that if you are creating a cross-region replica of an encrypted database you will also need to specify a `kms_key_id`."
+  default     = "changeme"
+}
+
+variable "rds_replica_kms_key_id" {
+  type        = "string"
+  description = "The ARN for the KMS encryption key. If creating an encrypted replica, set this to the destination KMS ARN."
+  default     = ""
+}
+
+# db.t2.micro is free tier
+# https://aws.amazon.com/rds/free
+variable "rds_replica_instance_type" {
+  type        = "string"
+  default     = "db.t2.micro"
+  description = "EC2 instance type for RDS DB"
+}
+
+variable "rds_replica_port" {
+  type        = "string"
+  default     = "3306"
+  description = "RDS DB port"
+}
+
+variable "rds_replica_snapshot" {
+  type        = "string"
+  default     = ""
+  description = "Set to a snapshot ID to restore from snapshot"
+}
+
+variable "rds_replica_multi_az" {
+  type        = "string"
+  default     = "false"
+  description = "Run instaces in multiple az"
+}
+
+variable "rds_replica_storage_type" {
+  type        = "string"
+  default     = "gp2"
+  description = "Storage type"
+}
+
+variable "rds_replica_storage_size" {
+  type        = "string"
+  default     = "20"
+  description = "Storage size"
+}
+
+variable "rds_replica_storage_encrypted" {
+  type        = "string"
+  default     = "true"
+  description = "Set true to encrypt storage"
+}
+
+variable "rds_replica_auto_minor_version_upgrade" {
+  type        = "string"
+  default     = "false"
```

Comment:
do we want to automatically allow minor version upgrade? (at least we did it before)